src/core/load-fragment.c

```diff
@@ -4765,6 +4765,47 @@ int config_parse_exec_directories(
         }
 }
 
+int config_parse_exec_quota(
+                const char *unit,
+                const char *filename,
+                unsigned line,
+                const char *section,
+                unsigned section_line,
+                const char *lvalue,
+                int ltype,
+                const char *rvalue,
+                void *data,
+                void *userdata) {
+
+        QuotaLimit *quota_limit = data;
+        uint64_t quota_absolute = UINT64_MAX;
+        uint32_t quota_scale = UINT32_MAX;
+        int r;
+
+        if (isempty(rvalue))
+                return 0;
+
+        r = parse_permyriad(rvalue);
+        if (r < 0) {
+                uint64_t bytes = CGROUP_LIMIT_MAX;
+                r = parse_size(rvalue, 1024, &bytes);
+                if (r < 0) {
+                        log_syntax(unit, LOG_WARNING, filename, line, r, "Failed to parse disk quota value, ignoring: %s", rvalue);
+                        return 0;
+                }
+                quota_absolute = bytes;
+        } else {
+                /* Normalize to 2^32-1 == 100% */
+                quota_scale = UINT32_SCALE_FROM_PERMYRIAD(r);
+        }
```

Comment:
no {} around single line if blocks.

Reply:
as mentioned elsewhere, please also support an explicit way to turn quota off via "off".